### DISCUSSION TITLE
ability to send ungrouped alert notifications

### DISCUSSION THREAD

**Discussion by matrog** [grafana/grafana#46515] [Alerting]
Hello team , I guess would be useful to have the option to send notifications (mail, slack, pd...) for every single alert and not for grouped alerts. In order to keep the number of notification policies low I would like to use something like

`alert_name =~ ^database_.*`

so it can match the following alerts

- alert_name = database_cpu
- alert_name = database_memory
- alert_name = disk_full

if I do that now and these alerts trigger at the same time I receive a notification which includes all of them with a title which doesn't tell me anything about the alert because these alerts may have different label and no one in common.

I would like to have 3 different emails or slack messages.

Same thing should be present for a single alert using "reduction" function so we have many instances of the same. I would to have a single notification for each instance.

in the end I would like to choose If I want to receive multiple message or grouped messages.

Thanks!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
